src/util/random-string.ts

```diff
@@ -1,7 +1,13 @@
 export const createRandomString = () => {
     const USE_HEX = 36;
     const SKIP_LEADING_ZERODOT = 2;
+    const ASCII_A = 97;
+    const NUMBER_OF_LETTERS = 26;
+
     return (
+        String.fromCharCode(
+            ASCII_A + Math.floor(Math.random() * NUMBER_OF_LETTERS)
+        ) +
```

Comment:
If we want to ensure the string starts with a letter, we could also just prepend an `a` to all strings generated…

Although that would fail if the consumer then crops that string to a single letter, but if they do that, there's a 1/26 chance of getting a duplicate on the second string anyway…

Reply:
I guess this way is more robust, and it's not like it's going to add masses of computing time 😄